actions/setup-token/src/main.ts

```diff
@@ -0,0 +1,119 @@
+import * as github from "@actions/github";
```

Comment:
We aren't very good about including the license headers. We could probably add a pre-submit check at some point.
```suggestion
/*
Copyright 2022 SLSA Authors

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    https://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WIHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/
import * as github from "@actions/github";
```